beancount/scripts/example.py

```diff
@@ -1836,7 +1836,13 @@ def convert(self, value, param, ctx):
 @click.option("--date-birth", type=LiberalDate(), help="Fictional date of birth.")
 @click.option("--seed", "-s", type=int, help="Random seed.")
 @click.option("--no-reformat", is_flag=True, help="Do not reformat the output.")
-@click.option("--output", "-o", type=click.File("w"), default="-", help="Output filename.")
+@click.option(
+    "--output",
+    "-o",
+    type=click.File("w", encoding="utf-8"),
+    default="-",
+    help="Output filename.",
+)
```

Comment:
Can we keep this on one line? This black-style formatting is really ugly.

Reply:
Can't. It reach current length limit.